SUPPORT.md

```diff
@@ -0,0 +1,8 @@
+Tor Support
```

Comment:
Please add a list of projects, and then put each project in its own section.

It's ok if the list just has "Tor (the network daemon)" for now.